packages/core-events/plugins/matomo-tracker.js

```diff
@@ -2,28 +2,31 @@ import fetch from 'isomorphic-unfetch';
 import { encode } from 'querystring';
 import { subscribe } from '../director';
 
-const extractOrderParameters = (orderPosition) => {
+const parseCurrency = (amount) => parseFloat(amount / 100).toFixed(2);
```

Comment:
Why you do a parseFloat here? Please use accounting package if you do divide and toFixed. Else we will have problems with the float handling of js (http://openexchangerates.github.io/accounting.js/)